web/components/NavBlock/NavBlock.tsx

```diff
@@ -0,0 +1,103 @@
+import Link from 'next/link';
```

Comment:
Could this file could be made shorter if you extract the various repeated elements into their own components, either directly in this file or in a subdirectory. With the former perhaps requiring less effort for sharing styles.

Example of what I'm imagining
``` 
const FakeItem = ({ mobile }: { mobile?: boolean }) =>
  <li
    className={clsx(
      styles.fakeItem,
      mobile && styles.mobile,
      styles.tablet,
      styles.desktop
    )}
    aria-hidden="true"
  />;
``` 

And then usage could simply be `<FakeItem mobile />` etc

(This is a first-glance suggestion, YMMV. I might have misjudged how many SLOC this will save.)

Reply:
Hm, I think that sounds like a really good idea. (And probably with a preference for having in the same file, yes.) And similarly for `<Divider … />`

I suspect you'd be faster at this than me, so I'll appreciate it if you can help out if there's time. For now I'll merge it as-is.